tests/testthat/test-commented_code_linter.R

```diff
@@ -92,15 +92,13 @@ test_that("commented_code_linter can detect operators in comments and lint corre
     expect_no_lint(paste("# something like i", op, "1", collapse = ""), linter)
     expect_lint(paste("# i", op, "1", collapse = ""), lint_msg, linter)
   }
-})
 
-test_that("commented_code_linter can detect operators in comments and lint correctly", {
   skip_if_not_r_version("4.1.0")
 
   expect_lint(
     "# 1:3 |> sum()",
     rex::rex("Remove commented code."),
-    commented_code_linter()
+    linter
```

Comment:
This failed to merge correctly

Reply:
Maybe do an interactive rebase of the most recent PR and update the preceding ones from there via force push?

Reply:
Only 3 more, I will just suffer through to avoid the `--force`